src/Components/QrScanner/QrScannerPlugin.tsx

```diff
@@ -25,7 +25,7 @@ export const calcAspectRatio = () => {
 export async function scanFile(file: File): Promise<string> {
   const scanner = new Html5Qrcode(qrcodeFileRegionId, {
     formatsToSupport: [Html5QrcodeSupportedFormats.QR_CODE],
-    verbose: process.env.NODE_ENV === 'development',
+    verbose: import.meta.env.NODE_ENV === 'development',
```

Comment:
I think you can use `import.meta.env.DEV` and `import.meta.env.PROD` (untested): https://vitejs.dev/guide/env-and-mode
```suggestion
    verbose: import.meta.env.DEV,
```

Reply:
Tested, works. Done!